contrib/test_jenkins.sh

```diff
@@ -888,11 +882,12 @@ test_profiling() {
 }
 
 test_ucs_load() {
-	if [ -z have_ptrace ]
+	if [ -z "${have_ptrace}" ]
 	then
-		echo "==== Not running UCS library loading test ===="
+		log_warning "==== Not running UCS library loading test ===="
```

Comment:
azure_log_warning
https://github.com/openucx/ucx/blob/6769bd348b0543c25affbd9e6f760a37ecd3a9d1/buildlib/az-helpers.sh#L50

Reply:
I think it's better to revert to `log_warning` - it uses `azure_log_warning` if needed, and in other places in this file `log_warning` is used.